src/main/java/adris/altoclef/Settings.java

```diff
@@ -609,6 +615,11 @@ public int getNetherFastTravelWalkingRange() {
     public BlockPos getHomeBasePosition() {
         return homeBasePosition;
     }
+    
+    
+	public boolean openInvDuringCrafting() {
+		return openInvDuringCrafting;
```

Comment:
This is odd to have in a task not related to crafting. Why is this here?

Reply:
I was adding a setting to altoclef to be able to toggle an option to allow altoclef to open inventory when crafting in 2x2 grid.

I thought this is the place where settings like this should be added.

Did I misunderstand the usage of this class?

I didn't find any other files that specify settings in altoclef.

This is what the setting makes altoclef do: https://www.youtube.com/watch?v=NUdvg6aFYss